skimage/filters/_rank_order.py

```diff
@@ -14,22 +14,21 @@
 def rank_order(image):
     """Return an image of the same shape where each pixel is the
     index of the pixel value in the ascending order of the unique
-    values of `image`, aka the rank-order value.
+    values of ``image``, aka the rank-order value.
 
     Parameters
     ----------
-    image: ndarray
+    image : ndarray
```

Comment:
Are these new spaces a thing?

Reply:
This is a fix to match the convention.

Reply:
people just make up standards. I guess it does follow the numpydoc documentation
https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard

but I would have thought that there should be no space, like a python dictionary.